.changeset/big-ghosts-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cartesi/cli": patch
+---
+
+enforce build for linux/riscv64 paltform
```

Comment:
typo

Reply:
even the branch name has a type :facepalm: 

I've sent a fix with push force to fix the type in the changeset and commit comment